posts/2021-07-28-GATs-stabilization-push.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "The push for GATs stabilization"
+author: Jack Huey
+team: the Traits Working Group <https://www.rust-lang.org/governance/teams/compiler#wg-traits>
+---
+
+# The push for GATs stabilization
+
+Where to start, where to start...
+
+Let's begin by saying: this is a very exciting post. Some people reading this will be overwhelmingly thrilled; some will have no idea what this feature is; others might be in disbelief. The [RFC] for this feature *did* get opened in April of *2016* (and merged about a year and a half later). In fact, this RFC even predates *const generics*. Don't let this fool you though: it is a powerful feature; and the reactions to the tracking issue on Github should maybe give you an idea of its popularity (it *is* the most upvoted issue on the Rust repository):
```

Comment:
Besides the mention in the title, the subject of this post only gets referred to as "this feature" in the entire first paragraph. Additionally, what the "GATs" feature is is not explained until after line 30. I would make a call out to "generic associated types" in the first paragraph and explain that if you're unfamiliar with what GATs are, we'll be explaining it below.

Reply:
I added a bit to address this. Does it look better now?